docs/index.rst

```diff
@@ -216,3 +216,72 @@ In dask-distributed, a ``Worker`` is a Python object and node in a dask
 computations. ``Jobs`` are resources submitted to, and managed by, the job
 queueing system (e.g. PBS, SGE, etc.). In dask-jobqueue, a single ``Job`` may
 include one or more ``Workers``.
+
+How to debug
+------------
+
+Dask jobqueue has been developed and tested by several contributors, each of
+us having a given HPC system setup to work on: a job scheduler in a given
+version running on a given OS. Thus, in some specific case, it might not work
+out of the box on your system. This section provides some hints to help you
+sort what may be going wrong.
+
+Checking job script
+~~~~~~~~~~~~~~~~~~~
+
+Often, the first thing to do is checking if the job script generated to submit
```

Comment:
Maybe start this section with something more like: 

Dask-jobqueue submits "job scripts" to your queueing system. Inspecting these scripts often reveals errors in the configuration of your Cluster.